bindings/rust/src/pippenger-no_std.rs

```diff
@@ -67,23 +67,20 @@ macro_rules! pippenger_mult_impl {
                     panic!("scalars length mismatch");
                 }
 
-                let p: [*const $point_affine; 2] =
-                    [&self.points[0], ptr::null()];
-                let s: [*const u8; 2] = [&scalars[0], ptr::null()];
+                let p = [self.points.as_ptr(), ptr::null()];
+                let s = [scalars.as_ptr(), ptr::null()];
 
+                let mut scratch =
+                    vec![0u64; unsafe { $scratch_sizeof(npoints) } / 8];
```

Comment:
The intent here is to avoid a useless zero initialization when ret.points will be initialized 5 lines below.